File: src/main/java/me/lokka30/levelledmobs/result/AdjacentChunksResult.java
package me.lokka30.levelledmobs.result;

import java.util.LinkedList;
import java.util.List;
import org.jetbrains.annotations.NotNull;

/**
 * Used in conjunction with the chunk kill count feature
 *
 * @author stumper66
 * @since 3.4.0
 */

public class AdjacentChunksResult {

    public AdjacentChunksResult() {
        this.chunkKeys = new LinkedList<>();
    }

    public int entities;
    public final @NotNull List<Long> chunkKeys;
}

File: levelledmobs-plugin/src/main/kotlin/io/github/arcaneplugins/levelledmobs/result/AdjacentChunksResult.kt
package io.github.arcaneplugins.levelledmobs.result

/**
 * Used in conjunction with the chunk kill count feature
 *
 * @author stumper66
 * @since 3.4.0
 */
class AdjacentChunksResult {
    var entities: Int = 0
    val chunkKeys = mutableListOf<Long>()
}